### DISCUSSION TITLE
Phillips A19 Dimmable Warm White Not Adding

### DISCUSSION THREAD

**Discussion by TheSquigglyLine** [sbidy/wiz_light#128] [Q&A]
I am going through the configuration process for this extension and I had no problem adding my LED Strips as lights but when I add my A19 Dimmable Warm White bulbs they do not show up in HA. I have checked the IP and syntax and don't believe that's the issue. The main page says that these bulbs are supported.
Where could/should I start looking to solve this problem?

**Comment by TheSquigglyLine** [comment]
I am able to add them as switches and then they work but obviously without any of the control over the brightness or warmth

**Comment by sbidy** [comment]
Ist seems to be the same issue with these types of bulbs. See #103 - I'm not able to reproduce but it the error comes from the dynamic effect mapping.
